client/vue.config.js

```diff
@@ -1,6 +1,9 @@
 // Read in .env file.
 const process = require('process');
-require('dotenv').config();
```

Comment:
I thought about doing this but I didn't like it for some reason. Maybe that it makes the NPM package refer outside itself to pick up configs, but I don't think that's really a problem since you generally have to build the client in context of this whole repo.

Thanks for this change.